recipes/bioconductor-a4base/meta.yaml

```diff
@@ -50,7 +50,5 @@ about:
   home: 'http://bioconductor.org/packages/{{ bioc }}/bioc/html/{{ name }}.html'
   license: GPL-3
   summary: 'Automated Affymetrix Array Analysis'
-extra:
```

Comment:
@dpryan79 please ignore the `extra` section and do not overwrite it. This section is manually added with curation and we probably would like to keep it.
Indeed it would be cool if we could extract this information at some point from the package with the skeleton.

Reply:
D'oh, I bet I accidentally did that when I removed the skip-lints stuff. I'll add them all back in tomorrow.